ElasticSearch/action_query_data.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "Query data",
+  "description": "Query data from ElasticSearch based on your configuration",
```

ElasticSearch allows querying the data with several languages (like [EQL](https://www.elastic.co/guide/en/elasticsearch/reference/8.17/eql-syntax.html), [ES|QL](https://www.elastic.co/guide/en/elasticsearch/reference/8.17/esql-getting-started.html) or [SQL](https://www.elastic.co/guide/en/elasticsearch/reference/8.17/sql-overview.html).
We have to indicate the expected language in the action.

```suggestion
  "name": "Execute ES|QL query",
  "description": "Query data from ElasticSearch with ES|QL",
```

I suggest renaming the docker_parameter and the Action class to reflect the language used.
In the future, maybe an action to query data with the EQL language will be created.